### PR TITLE
Send heartbeat after one minute if "receivedAnyStats: false"

### PR DESCRIPTION
**Pull request by bitterpanda63**

### CODE REVIEW

File: aikido_firewall/background_process/reporter/__init__.py [hunk 1]
```diff
@@ -49,11 +50,23 @@ def __init__(self, block, api, token, serverless):
...
             self.update_service_config, self.serverless, self.token, event_scheduler
         )
 
+    def report_initial_stats(self):
+        """
+        This is run 1m after startup, and checks if we should send out
+        a preliminary heartbeat with some stats.
+        """
+        should_report_initial_stats = not (
+            self.statistics.is_empty() or self.conf.received_any_stats
```
Comment (hansott) [inline aikido_firewall/background_process/reporter/__init__.py:75]: `self.statistics.is_empty()` what if there's a route or user? then we won't send any stats?

Comment (bitterpanda63) [reply aikido_firewall/background_process/reporter/__init__.py:75]: In node rn it's only sinks we are looking at right? But if there is a route then the requests total would've gone up so we would report that

File: aikido_firewall/background_process/reporter/__init__.py [hunk 2]
```diff
@@ -26,14 +26,15 @@ class Reporter:
...
+    initial_stats_timeout = 60  # Wait 60 seconds after startup for initial stats
 
     def __init__(self, block, api, token, serverless):
         self.block = block
         self.api = api
         self.token = token  # Should be instance of the Token class!
         self.routes = Routes(200)
         self.hostnames = Hostnames(200)
-        self.conf = ServiceConfig([], get_unixtime_ms(), [], [])
+        self.conf = ServiceConfig([], get_unixtime_ms(), [], [], True)
```
Comment (willem-delbare) [inline aikido_firewall/background_process/reporter/__init__.py]: can we use named params here? hard to figure out what 'true' does

Comment (bitterpanda63) [reply aikido_firewall/background_process/reporter/__init__.py]: Done

**Pull request merged**